src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,211 @@
+package duke;
+import java.io.*;
```

Comment:
Please avoid using wildcard imports. Consider importing each class you require instead.